client/tracing/proc-macro/src/lib.rs

```diff
@@ -120,16 +120,17 @@ pub fn prefix_logs_with(arg: TokenStream, item: TokenStream) -> TokenStream {
 
 	let crate_name = if std::env::var("CARGO_PKG_NAME")
 		.expect("cargo env var always there when compiling; qed")
-		== "sc-cli"
+		== "sc-tracing"
 	{
-		Ident::new("sc_cli", Span::call_site().into())
+		Path::from(Ident::new("sc_tracing", Span::call_site()))
 	} else {
-		let crate_name = match crate_name("sc-cli") {
-			Ok(x) => x,
-			Err(err) => return Error::new(Span::call_site(), err).to_compile_error().into(),
-		};
-
-		Ident::new(&crate_name, Span::call_site().into())
+		match crate_name("sc-cli") {
```

Comment:
I think we don't need this re-export.

Reply:
Ok I will remove it then. I just made it because it was handy to have it on sc-cli but I'm really unsure about this. It's not a big deal to important sc-logging after all